python-threatexchange/threatexchange/extensions/video_vpdq/tests/test_vpdq_hash_and_match.py

```diff
@@ -0,0 +1,31 @@
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
+import pytest
+from pathlib import Path
+import sys
+
+sys.path.append(str(Path(__file__).parents[1]))
```

Comment:
blocking: This looks dangerous, and breaks a general rule of python that imports should not have side effects. What are the alternatives to do doing this / what problem does this solve?